src/plugins/score-plugin-remotecontrol/CMakeLists.txt

```diff
@@ -41,12 +42,16 @@ set(SRCS
 "${CMAKE_CURRENT_SOURCE_DIR}/RemoteControl/ApplicationPlugin.cpp"
 "${CMAKE_CURRENT_SOURCE_DIR}/RemoteControl/DocumentPlugin.cpp"
 
+"${CMAKE_CURRENT_SOURCE_DIR}/RemoteControl/Http_server.cpp"
+
 "${CMAKE_CURRENT_SOURCE_DIR}/score_plugin_remotecontrol.cpp"
 )
 
+file(COPY ${CMAKE_CURRENT_SOURCE_DIR}/RemoteControl/build-wasm/ DESTINATION ${CMAKE_BINARY_DIR}/src/plugins/score-plugin-remotecontrol/CMakeFiles/score_plugin_remotecontrol.dir/RemoteControl/build-wasm/)
```

Comment:
we won't need that: the remote's .wasm file is not supposed to be built as part of score but in a different repo which will be in the package manager